src/migrations/m230217_100801_add_taxjar_categories_prefix.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace craft\commerce\taxjar\migrations;
+
+use craft\commerce\taxjar\services\Api;
+use craft\db\Migration;
+use craft\db\Query;
+
+/**
+ * m230217_100801_add_taxjar_categories_prefix migration.
+ */
+class m230217_100801_add_taxjar_categories_prefix extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public function safeUp(): bool
+    {
+        $taxCategories = (new Query())
+            ->select(['id', 'handle'])
+            ->from(['taxcategories' => \craft\commerce\db\Table::TAXCATEGORIES])
+            ->all();
+
+        foreach ($taxCategories as $taxCategory) {
+            $handle = $taxCategory['handle'];
+            $id = $taxCategory['id'];
+            $firstChar = substr($handle, 0, 1);
+
+            if (is_numeric($firstChar)) {
```

Comment:
Do we need to test if it's numeric? I would expect to have all tax jar categories to get the prefix even if they start with a letter.

Reply:
Yes, because there could be a tax category before they installed the plugin. For instance, there is a `General` tax category. If I don’t check if it is numeric, It will add the prefix to the `General` handle.